metricbeat/Dockerfile

```diff
@@ -15,5 +17,14 @@ RUN pip3 install --upgrade pip
 RUN pip3 install --upgrade setuptools
 RUN pip3 install --upgrade docker-compose==1.23.2
 
+# Oracle instant client
+RUN cd /usr/lib \
+  && curl -sLo instantclient-basic-linux.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip \
+  && unzip instantclient-basic-linux.zip \
+  && rm instantclient-basic-linux.zip
+ENV LD_LIBRARY_PATH=/usr/lib/instantclient_19_6
```

Comment:
So at the end we don't depend on another docker image?

Reply:
no, the latest version doesn't require to sing a form